Makefile

```diff
@@ -220,10 +220,9 @@ $(BIN_DIR):
 $(LIB_DIR):
 	mkdir -p $@
 
-$(MEMCACHED_BIN): | $(BIN_DIR) $(LIB_DIR)
-	@echo "Downloading Memcached"
-	curl -L https://archive.org/download/archlinux_pkg_libevent/libevent-2.1.12-1-x86_64.pkg.tar.zst | tar --strip-components=2 --use-compress-program=unzstd -xf - -C $(LIB_DIR) usr/lib
-	curl -L https://archive.org/download/archlinux_pkg_memcached/memcached-1.6.18-1-x86_64.pkg.tar.zst | tar --strip-components=2 --use-compress-program=unzstd -xf - -C $(BIN_DIR) usr/bin/memcached
+$(MEMCACHED_BIN): 
+	@echo "test dnf" 
+	dnf install memcached-1.6.21-2.fc39.x86_64
```

Comment:
don't install in that way. I'd leave the specific version out as that may slightly change in the future and at least add user `dnf install -y --setopt=tsflags=nodocs`

Otherwise it'll require confirmation and most likely fail and also pull unused docs and waste space.

Reply:
done